config/routes.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require "sidekiq/web"
```

Comment:
```suggestion
require "sidekiq/pro/web"
```

Reply:
...but for local development only, omit the `/pro`

Reply:
done